### PR TITLE
Round business turnover to 2 significant figures

### PR DESCRIPTION
**Pull request by paulgain**
## Description of change
Formats company turnover in USD to GBP by doing two distinct things:
1. Converts USD to GBP using a hardcoded (for now) USDGBP currency exchange rate.
2. Takes the GPB (from no.1) and rounds the value to 2 significant figures.

Follows on from this: https://github.com/uktrade/data-hub-components/pull/260

## Test instructions
**/UUID/business-details**

_Here are 6 examples:_
| Company name                          | URL                                                                                                                  | Turnover USD | USDGBP SPOT | Turnover GBP | Rounded Turnover GBP (2 sig. fig.) |
|---------------------------------------|----------------------------------------------------------------------------------------------------------------------|--------------|-----------------|-------------|------------------|
| The Troll Company Ltd                 | https://data-hub-fro-annual-tur-kpnhoz.herokuapp.com/companies/a2777eab-7336-4e16-845d-1be2cac10d37/business-details | 1092003      | 0.7678          | 838439.9034 |  840,000         |
| 2ND MAGPIE LTD                        | https://data-hub-fro-annual-tur-kpnhoz.herokuapp.com/companies/fc988eea-43fb-4ced-87e6-71ce64dc77c1/business-details | 88056        | 0.7678          | 67609.3968  |  68,000          |
| RAF LIMITED                           | https://data-hub-fro-annual-tur-kpnhoz.herokuapp.com/companies/66b6516e-fd19-476b-ab05-3e40fd0ba457/business-details | 43439828     | 0.7678          | 33353099.94 |  33,000,000      |
| COLOUR MATCH PRINT MANAGEMENT LIMITED | https://data-hub-fro-annual-tur-kpnhoz.herokuapp.com/companies/36bb6bf5-c215-496b-b535-b6f435c4cf8b/business-details | 1415899      | 0.7678          | 1087127.252 |  1,100,000       |
| JOHN BULL (CONFECTIONERS) LIMITED     | https://data-hub-fro-annual-tur-kpnhoz.herokuapp.com/companies/1a329afd-0c06-4c36-823d-018a29e1edcb/business-details | 15951252     | 0.7678          | 12247371.29 |  12,000,000      |
| The Walt Disney Company               | https://data-hub-fro-annual-tur-kpnhoz.herokuapp.com/companies/c459169e-8005-4df3-9ba2-f31463ec4e5a/business-details | 69570000000  | 0.7678          | 53415846000 |  53,000,000,000  |

### Before
before

### After
after

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/companies/apps/business-details/client/SectionAbout.jsx [hunk 1]
```diff
@@ -69,7 +69,9 @@ const SectionAbout = ({ businessDetails, isDnbCompany, isArchived, urls }) => (
     <SummaryTable.Row heading="Annual turnover">
       {businessDetails.turnover && (
         <>
-          {NumberUtils.currencyGBP(businessDetails.turnover)}
+          {NumberUtils.currencyGBP(businessDetails.turnover, {
+            maximumSignificantDigits: 2,
```
Comment (rafenden) [inline src/apps/companies/apps/business-details/client/SectionAbout.jsx]: No need for the second argument as the default one is exactly the same.

File: src/apps/companies/apps/edit-history/client/EditHistoryChangeList.jsx [hunk 2]
```diff
@@ -60,7 +60,9 @@ function getValue(value, field) {
 
   if (isNumber(value)) {
     return CURRENCY_FIELDS.includes(field)
-      ? NumberUtils.currencyGBP(convertUSDToGBP(value))
+      ? NumberUtils.currencyGBP(convertUSDToGBP(value), {
+          maximumSignificantDigits: 2,
```
Comment (rafenden) [inline src/apps/companies/apps/edit-history/client/EditHistoryChangeList.jsx]: No need for the second argument as the default one is exactly the same.

File: src/apps/companies/apps/business-details/client/SectionAbout.jsx (hunk 1 above)
Comment (paulgain) [reply src/apps/companies/apps/business-details/client/SectionAbout.jsx]: No, the default is `21`.

File: src/apps/companies/apps/edit-history/client/EditHistoryChangeList.jsx (hunk 2 above)
Comment (paulgain) [reply src/apps/companies/apps/edit-history/client/EditHistoryChangeList.jsx]: No, the default is `21`.

File: src/common/currency.js [hunk 3]
```diff
@@ -1,18 +1,12 @@
 const { isNumber } = require('lodash')
-const { CURRENCY_RATE_USD_TO_GBP } = require('./constants')
+const { EXCHANGE_RATE_USD_TO_GBP } = require('./constants')
 
 const convertUSDToGBP = (usd) => {
```
Comment (web-bert) [inline src/common/currency.js]: Optional: this should be `convertUsdToGbp` to adhere to camel case

Comment (paulgain) [reply src/common/currency.js]: This is consistent with the rest of our codebase. Here are 3 examples:
1. https://github.com/uktrade/data-hub-frontend/blob/master/src/apps/companies/apps/activity-feed/constants.js
2. https://github.com/uktrade/data-hub-frontend/blob/master/src/apps/companies/apps/add-company/client/constants.js
3. https://github.com/uktrade/data-hub-frontend/blob/master/src/apps/companies/apps/edit-history/constants.js

"Constants can be declared with uppercase or lowercase, but a common convention is to use all-uppercase letters."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#Examples

Comment (paulgain) [reply src/common/currency.js]: 👍

**Pull request merged**